<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloadWidget</class>
 <widget class="QWidget" name="MusicDownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下载窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>460</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>下载窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>460</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:white;}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>保存名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:22px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="MusicDownloadTableWidget" name="viewArea">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>33</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>37</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>资源选择：</string>
    </property>
   </widget>
   <zorder>label2</zorder>
   <zorder>downloadName</zorder>
   <zorder>viewArea</zorder>
   <zorder>label1</zorder>
   <zorder>fileNameLabel</zorder>
   <zorder>downloadButton</zorder>
  </widget>
  <widget class="MusicGifLabelMaskWidget" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>460</width>
     <height>195</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicDownloadTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicdownloadwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicGifLabelMaskWidget</class>
   <extends>QLabel</extends>
   <header>musicgiflabelwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
